flanker-test-app/src/main/java/edu/northwestern/mobiletoolbox/flanker/test_app/FlankerTestApplication.java

```diff
@@ -0,0 +1,23 @@
+package edu.northwestern.mobiletoolbox.flanker.test_app;
+
+import com.jakewharton.threetenabp.AndroidThreeTen;
+
+import dagger.android.AndroidInjector;
+import dagger.android.support.DaggerApplication;
+import edu.northwestern.mobiletoolbox.flanker.test_app.inject.DaggerFlankerTestAppComponent;
+
+public class FlankerTestApplication extends DaggerApplication {
+    @Override
+    public void onCreate() {
+        AndroidThreeTen.init(this);
+        super.onCreate();

```

Comment:
why do you call the super method in the end? 
As I know it has to be at the beginning of method.

Reply:
I moved super to first call, you're right, this is usually how it's done.

The reason for this is super triggers Dagger injection, and some of our apps use libraries that do time operations upon initialization. But moving it for this app should be fine